src/class-wp-plugin-loader.php

```diff
@@ -265,7 +265,7 @@ public function filter_plugin_action_links( $actions, $plugin_file ): array {
 	 * @return array<int, string>
 	 */
 	public function filter_option_active_plugins( $value ): array {
-		if ( ! is_array( $value ) ) {
+		if ( ! is_array( $value ) ) { // @phpstan-ignore-line to true
```

Comment:
Curious - why is phpstan complaining here?

Reply:
PHPDoc says it's an array but I never trust a filter's arguments.

Reply:
Ah, makes sense. Maybe don't label it as an array in the PHPDoc then? Explicitly state `mixed` because it could be something else, then you're being defensive and treating it as an unknown?